.ycm_extra_conf.py

```diff
@@ -5,13 +5,13 @@
 
 CONDA_PREFIX = os.environ["CONDA_PREFIX"]
 NVJITLINK_DIR = Path(__file__).parent
-VER = sys.version_info
-PYTHON_INCLUDE_NAME = "python3.10"  # Not ideal
+PYTHON_VERSION = sys.version_info
+PYTHON_INCLUDE_NAME = "python%d.%d" % PYTHON_VERSION
```

Comment:
This could be a more robust solution

```suggestion
PYTHON_INCLUDE_NAME = sysconfig.get_config_var("INCLUDEPY")
```

Reply:
This would need to be assigned to `PYTHON_INCLUDE_DIR` below because it gives the full path.